homeassistant/components/media_player/plex.py

```diff
@@ -418,65 +421,53 @@ def refresh(self, device, session):
             self._media_content_type = MEDIA_TYPE_VIDEO
         elif self._session_type == 'track':
             self._media_content_type = MEDIA_TYPE_MUSIC
-        else:
-            self._media_content_type = None
 
         # title (movie name, tv episode name, music song name)
-        if self._session:
+        if self._session and self._is_player_active:
             self._media_title = self._convert_na_to_none(self._session.title)
 
         # Movies
         if (self.media_content_type == MEDIA_TYPE_VIDEO and
                 self._convert_na_to_none(self._session.year) is not None):
             self._media_title += ' (' + str(self._session.year) + ')'
 
-        # TV Show
-        if (self._is_player_active and
-                self._media_content_type is MEDIA_TYPE_TVSHOW):
-
-            # season number (00)
-            if callable(self._convert_na_to_none(self._session.seasons)):
-                self._media_season = self._convert_na_to_none(
-                    self._session.seasons()[0].index).zfill(2)
-            elif self._convert_na_to_none(
-                    self._session.parentIndex) is not None:
-                self._media_season = self._session.parentIndex.zfill(2)
-            else:
-                self._media_season = None
-
-            # show name
-            self._media_series_title = self._convert_na_to_none(
-                self._session.grandparentTitle)
-
-            # episode number (00)
-            if self._convert_na_to_none(
-                    self._session.index) is not None:
-                self._media_episode = str(self._session.index).zfill(2)
-        else:
-            self._media_season = None
-            self._media_series_title = None
-            self._media_episode = None
-
-        # Music
-        if (self._is_player_active and
-                self._media_content_type == MEDIA_TYPE_MUSIC):
-            self._media_album_name = self._convert_na_to_none(
-                self._session.parentTitle)
-            self._media_album_artist = self._convert_na_to_none(
-                self._session.grandparentTitle)
-            self._media_track = self._convert_na_to_none(self._session.index)
-            self._media_artist = self._convert_na_to_none(
-                self._session.originalTitle)
-            # use album artist if track artist is missing
-            if self._media_artist is None:
-                _LOGGER.debug("Using album artist because track artist was "
-                              "not found: %s", self.entity_id)
-                self._media_artist = self._media_album_artist
-        else:
-            self._media_album_name = None
-            self._media_album_artist = None
-            self._media_track = None
-            self._media_artist = None
+        if self._is_player_active:
```

Comment:
Reading through the code, it looks to me like `self._media_content_type` will only be not None if `self._is_player_active` is True, so I think we can drop this outer if all-together. It would be good if that's the case to simplify this diff.

Reply:
I'll do that today/tonight was away at a conf so no access to test. I had left it in because I wasnt sure on impact and and I want to clean up the working code 1st, i think a few line above that there was then entry for Movies so I did think that if was possibly to be removed else the Movies part was to come under that.. 

Also I've noticed in my case the IF SESSION: at the top is (seemingly) never triggered but if DEVICE is always triggered. So I'm not sure in which case it would get called.